src/raygun.rum/index.js

```diff
@@ -1164,7 +1177,7 @@ var raygunRumFactory = function (window, $, Raygun) {
     function shouldIgnoreResource(resource) {
       var name = resource.name.split('?')[0];
 
-      return shouldIgnoreResourceByName(name) || resource.entryType === "paint" || resource.entryType === "navigation" || resource.entryType === "mark";
+      return shouldIgnoreResourceByName(name) || resource.entryType === "paint" || resource.entryType === "mark"|| resource.entryType === "navigation" || resource.entryType === "visibility-state"; //We dont want either the old or the new timings to be used for secondary, they should only be primary
```

Comment:
maybe a WIP, but the comment here should be removed. Consider adding this above to explain the `entryType` checks. comments will be stripped from the `.min.js` at build time anyway